lib/swaps/Swaps.ts

```diff
@@ -503,7 +504,7 @@ class Swaps extends EventEmitter {
       }
     }
 
-    const swapClient = this.swapClients.get(deal.makerCurrency);
+    const swapClient = this.swapClientManager.get(deal.makerCurrency);
     if (!swapClient) {
```

Comment:
I think we can just remove this if statement (it will never be falsey), if don't have the swap client after we'd already checked it earlier then it's probably fine to let the exception bubble up.